scripts/copy-ui-files.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./node_modules/.bin/ncp ./packages/neuron-ui/build ./packages/neuron-wallet/dist/neuron-ui
```

Comment:
Can we use link instead of copy ?

Reply:
I don’t think packaging process can handle link gracefully.

Reply:
OK, it seems only for package and e2e.